note-service/src/main/java/ma/enset/noteservice/controller/NoteElementController.java

```diff
@@ -61,11 +63,11 @@ public ResponseEntity<NoteElementWithElementResponse> get(@PathVariable("noteEle
                 .body(foundModuleResponse);
     }
     @GetMapping
-    public  ResponseEntity<NoteElementPagingResponse> getAll(@RequestParam(defaultValue = "0") @Min(0) int page,
-                                                            @RequestParam(defaultValue = "10") @Range(min = 1, max = 10) int size) {
+    public  ResponseEntity<NoteElementPagingResponse> getAllByCodeSession(@RequestParam(defaultValue = "0") @Min(0) int page,
+                                                            @RequestParam(defaultValue = "10") @Range(min = 1, max = 10) int size, @RequestParam String codeSession) {
```

Comment:
as said before, there is no need for pagination in this service.

Reply:
Sorry I forgot it